src/main/java/com/foo/gosucatcher/domain/member/presentation/MemberController.java

```diff
@@ -108,9 +147,9 @@ public ResponseEntity<Resource> findProfileImage(@PathVariable long memberId) {
 			.body(resource);
 	}
 
-	@DeleteMapping("/{memberId}/profile")
+	@DeleteMapping("/me/profile/image")
 	public ResponseEntity<Void> deleteProfileImage(@PathVariable long memberId) {
-		memberService.removeProfileImage(memberId);
+		memberService.deleteProfileImage(memberId);
 
 		return ResponseEntity.ok(null);
 	}
```

Comment:
로그아웃은 noContent로 반환해주셨던데 통일해주시는게 좋을 것 같아요
```suggestion
	public ResponseEntity<Object> deleteProfileImage(@PathVariable long memberId) {
		memberService.deleteProfileImage(memberId);

		return ResponseEntity.noContent().build();
	}
```

Reply:
수정하겠습니다~!